Sources/MapboxNavigation/CameraController.swift

```diff
@@ -134,8 +134,18 @@ class CameraController: NavigationComponent, NavigationComponentDelegate {
         let courseViewMinimumInsets = UIEdgeInsets(top: 75.0, left: 75.0, bottom: 75.0, right: 75.0)
         var insets = navigationMapView.mapView.safeArea
         insets += courseViewMinimumInsets
-        insets.top += topBannerContainerView.bounds.height
-        insets.bottom += bottomBannerContainerView.bounds.height + 10.0
+        
+        switch navigationViewData.navigationView.traitCollection.verticalSizeClass {
+        case .unspecified:
+            fallthrough
+        case .regular:
+            insets.top += topBannerContainerView.bounds.height
+            insets.bottom += bottomBannerContainerView.bounds.height + 10.0
+        case .compact:
+            insets.left += navigationViewData.navigationView.topBannerContainerView.frame.width + 10.0
```

Comment:
The leading and trailing constraints below are direction-independent, so the flip in a right-to-left language environment. Does that affect the insets required here?

Reply:
Good catch, improved current insets for the right-to-left languages. Interestingly enough Google Maps for iOS doesn't flip top and bottom banner views, whereas Apple Maps flip.